docs/bonding-intro.md

```diff
@@ -0,0 +1,255 @@
+What are groups
+===============
+
+A Group is an entity that binds multiple sockets and it is required to
+establish a "bonded connection". Groups can be then used the same way as
+sockets for performing a transmission. It is then in general stated that a
+group is connected as long as at least one member-socket connection is alive,
+and as long as this state lasts, some member connections may get broken and
+new member connections can be established.
```

Comment:
A Group is an entity that binds multiple sockets, and is required to
establish a "bonded connection". Groups can be used in the same way as
sockets for performing a transmission. A group is connected as long as at 
least one member-socket connection is alive. As long as a group is in the 
connected state, some member connections may be broken and
new member connections can be established.

Reply:
"may get broken" was written intentionally. The point is that they can transit from "connected" to "broken" (previously alive connections may break at some point).